sys/linux/video4linux.txt

```diff
@@ -455,7 +455,7 @@ v4l2_crop {
 v4l2_jpegcompression {
 	quality		int32
 	APPn		int32[0:15]
-	APP_len		len[APPn, int32]
+	APP_len		len[APP_data, int32]
 	APP_data	array[int8, 60]
 	COM_len		len[COM_data, int32]
```

Comment:
I don't how syzkaller handle this. Maybe APP_len and COM_len should be described as ```APP_len  int32[0:60]```. Clarification required.

Reply:
Yes, I think describing them as `int32[0:60]` is better.